src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -524,6 +528,22 @@ public string ConstructKeySubjectName(string topic, string recordType = null)
         public string ConstructValueSubjectName(string topic, string recordType = null)
             => valueSubjectNameStrategy(new SerializationContext(MessageComponentType.Value, topic), recordType);
 
+        /// <inheritdoc />
+        public async Task<Compatibility> GetCompatibilityAsync(string subject) 
+            => await restService.GetCompatibilityAsync(subject);
+
+        /// <inheritdoc />
+        public async Task<Compatibility> GetGlobalCompatibilityAsync() 
+            => await restService.GetGlobalCompatibilityAsync();
+
+        /// <inheritdoc />
+        public async Task SetCompatibilityAsync(string subject, Compatibility compatibility) 
+            => await restService.SetCompatibilityAsync(subject, compatibility);
+
+        /// <inheritdoc />
+        public async Task SetGlobalCompatibilityAsync(Compatibility compatibility) 
+            => await restService.SetGlobalCompatibilityAsync(compatibility);
```

Comment:
Same as above for merging, and renamed to `UpdateComatibilityAsync`. 

Also, I think we should return Compatibility to keep it aligned with the Java API https://github.com/confluentinc/schema-registry/blob/2d1127f05799d1e60f6702e246cca80a7061e50a/client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClient.java#L232-L235